regression/cbmc/pragma_cprover1/main.c

```diff
@@ -0,0 +1,15 @@
+int main()
+{
+  int x;
+  int y[1];
+
+#pragma CPROVER(push, no_bounds_check)
+#pragma CPROVER(push, no_signed_overflow_check)
+  // do not generate assertions for the following statement
+  x = x + y[1];
+  // pop all annotations
+#pragma CPROVER()
```

Comment:
no pop?

Reply:
There is `#pragma CPROVER(pop)` to pop a single annotation or `#pragma CPROVER()` to pop all of them. This info is included in the API documentation.